src/components/alert/alert.tsx

```diff
@@ -153,6 +153,8 @@ export class Alert implements OpenCloseComponent, LocalizedComponent, LoadableCo
       this.calciteInternalAlertRegister.emit();
     }
     connectOpenCloseComponent(this);
+    this.isSlottedInShell =
```

Comment:
Can't the `shell` just add some styling in its CSS like...

```
::slotted(alert) {
  @apply absolute;
}
```

Reply:
Alert is a simpler example, but the Modal required some internal configuration and I tried to keep the pattern consistent between the two. I also tried to keep the styles off of the Alert host and applied to an internal container, but I can re-evaluate this.

Reply:
Yeah. In theory your logic should be fine if this is the only way. Its just we don't currently do this anywhere else.

Reply:
Maybe we could introduce a property if necessary? Could be an internal property that only shell knows about.

Its just better if the parent controls its children rather than the children having to figure out their parent and do things.

Reply:
@driskull I updated the pattern to use internal properties on `Alert` and `Modal`, which is set by the parent `Shell` - let me know if this looks better.

Reply:
@driskull I updated the pattern to use internal properties on Alert and Modal, set by the parent Shell - let me know if this looks better.